modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -1,12 +1,14 @@
 include::{partialsdir}/attributes.adoc[]
 
-= Authenticating
+= User authentication
```

Comment:
This sections comes directly from "Adding Core", therefore I think we should add a "prerequisites" paragraph first with
[x] - You have imported the necessary libraries and initialized the SDK

Reply:
The procedure should describe what the user is trying to achieve,
Would 'Adding User Authentication to your Mobile App' make sense?